mozilla-release/mobile/android/base/strings.xml.in

```diff
@@ -656,7 +656,7 @@
   <string name="pref_what_is_human_web">&pref_what_is_human_web;</string>
   <string name="pref_enable_human_web">&pref_enable_human_web;</string>
   <string name="pref_category_human_web_summary">&pref_category_human_web_summary;</string>
-  <string name="pref_human_web_url">https://cliqz.com/&formatS1;/whycliqz/human-web</string>
+  <string name="pref_human_web_url">https://cliqz.com/en/whycliqz/human-web</string>
```

Comment:
1. Do not hardcode urls in strings.xml.in.
2. Use the url without the locale here, the website will detect the language automatically
3. Put the url inside the branding folder (brand.dtd), we will probably have to use a different url for Ghostery